### ISSUE TITLE
interface support

### ISSUE THREAD

**Issue by KFKMan** [BlackWhiteYoshi/InlineComposition#1]
Your project is very good.
Can you add interface support?

Example;

public class BaseA : IBaseA{}
public clas BaseB : IBaseB{}

[InLine<BaseA, BaseB>]
public partial class Entity{}
...

Generated Code;

public partial class Entity : IBaseA, IBaseB{}

**Comment by KFKMan** [comment]
And can you change API like;
InlineBase

InlineWithBaseNeed => if base types has no InlineBase ignore them

Inline => no ignoring, no need to inlinebase Attribute on base types

**Comment by BlackWhiteYoshi** [comment]
With all due respect, but please, can you put a little more effort in your requests. The spelling mistakes in your example aside, it is hard to follow what you actually want.

Your first request is about interfaces, but I do not know what exactly your problem is.
Maybe you just forgot to add *[InlineBase]* on *BaseA* and *BaseB*? When you decorate both classes with that attribute, then Entity will implement both interfaces.
If you mean something else, please explain in more detail.

Your second request is that *[Inline<>]* is working without the need of *[InlineBase]*. I do not want to go to technical, but let me tell you that will not be easy. It is possible, but it will be a big change. But feel free to create a branch, implement it and make a PR.
